src/main/java/com/semmle/jira/addon/LgtmServlet.java

```diff
@@ -168,6 +175,20 @@ void createIssue(
         writeErrors(issueResult, resp);
         return;
       }
+
+      CustomField customField =
```

Comment:
Please add a bit more error handling. For example `CUSTOM_FIELD_CONFIG_KEY` may not exist, it may not be a valid number (NumberFormatException). The custom field might not exist. Finally I think `updateValue` may throw some exceptions. There is nothing you can do about these error conditions, other than logging an error message and returning a `500` HTTP status code.